src/Pages/jiminkim/Main/Feeds/Comments/Comments.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import CommentList from './CommentList';
+import CommentInput from './CommentInput';
+
+class Comments extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentsList: [],
+      inputValue: '',
+      isInsert: false,
+    };
+  }
+
+  // 댓글 조회
+  componentDidMount() {
+    this.setState({
+      commentsList: this.props.commentsList,
+    });
+  }
+
+  // 댓글 Input 입력 Event
+  setCommentInput = event => {
+    if (event.key === 'Enter') {
+      if (event.target.value.replace(/(?:\r\n|\r|\n)/g, '') === '') return;
+      this.appendComment();
+      return;
+    }
+    this.setState({
+      inputValue: event.target.value,
+      isInsert: event.target.value !== '' ? true : false,
+    });
+  };
+
+  // 댓글 추가 Event
+  appendComment = () => {
+    this.setState({
+      commentsList: this.state.commentsList.concat({
+        no: this.state.commentsList.length + 1,
+        nickname: 'nickname',
+        comment: this.state.inputValue,
+        isHeart: false,
+      }),
```

Comment:
```suggestion
      commentsList: [...commentsList, {
        no: commentsList.length + 1,
        nickname: 'nickname',
        comment: inputValue,
        isHeart: false,
      }),
```
스프레드 연산자를 사용하셔도 동일한 결과를 얻을 수 있을 것 같습니다!

Reply:
스프레드 연산자로 변경했습니다 👍